pkg/storaged/stratis-details.jsx

```diff
@@ -762,22 +763,23 @@
         });
     }
 
-    return manager.client.call(manager.path, "org.storage.stratis2.FetchProperties.r2", "GetProperties", [["KeyList"]])
-            .catch(() => [{ }])
-            .then(([result]) => {
-                let keys = [];
-                if (result.KeyList && result.KeyList[0])
-                    keys = result.KeyList[1].v;
-                if (keys.indexOf(key_desc) >= 0)
-                    return unlock();
-                else
-                    unlock_with_keydesc(key_desc);
-            });
+    if (!key_desc) {
+        return start();
+    } else {
+        return (client.stratis_list_keys()
+                .catch(() => [{ }])
+                .then(keys => {
+                    if (keys.indexOf(key_desc) >= 0)
+                        return start("keyring");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18679-20230424-131415-eca64e0e-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)